src/shared/pages/welcomePage/localization.js

```diff
@@ -4,23 +4,30 @@ export function replaceMessage(element, l10nID, href) {
     return false;
   }
 
-  message = message.replace("<a>", `<a id="${l10nID}Link" href="">`);
-
-  // eslint-disable-next-line no-unsanitized/property
-  element.innerHTML = message;
+  if (message.includes("<a>")) {
+    const prefix = message.split("<a>")[0];
+    const suffix = message.split("</a>")[1];
+    const anchor = document.createElement("a");
+    anchor.id = `${l10nID}Link`;
+    anchor.href = href || "";
```

Comment:
In this case it doesn't matter (because extension CSP blocks inline scripts), but in general it would be recommended to verify that the user-defined link is NOT a javascript:-URL to avoid XSS issues. That is, sanitize or drop the URL if `anchor.protocol == "javascript:"`